### ISSUE TITLE
Loki clients fail to autodetect some network settings

### ISSUE THREAD

**Issue by KeeJef** [loki-project/loki-network#1009]
had to change my ini file to point to ifaddr=10.10.0.1/16, this should have been handled automatically

**Comment by majestrate** [comment]
remind me what network configuration caused this (and for future references for anyone viewing the issue)

**Comment by KeeJef** [comment]
```Ethernet adapter Ethernet:

   Connection-specific DNS Suffix  . : gateway
   Link-local IPv6 Address . . . . . : fe80::58d1:e511:c5c0:418f%20
   IPv4 Address. . . . . . . . . . . : 10.0.0.3
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : 10.0.0.138

Ethernet adapter VirtualBox Host-Only Network:

   Connection-specific DNS Suffix  . :
   Link-local IPv6 Address . . . . . : fe80::6c48:99bf:926c:563e%3
   IPv4 Address. . . . . . . . . . . : 192.168.56.1
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . :

Ethernet adapter Ethernet 3:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Ethernet adapter Npcap Loopback Adapter:

   Connection-specific DNS Suffix  . :
   Link-local IPv6 Address . . . . . : fe80::6816:8531:50c1:9c3d%19
   Autoconfiguration IPv4 Address. . : xxxxxxxxx
   Subnet Mask . . . . . . . . . . . : 255.255.0.0
   Default Gateway . . . . . . . . . :

Ethernet adapter Ethernet 2:

   Connection-specific DNS Suffix  . :
   Link-local IPv6 Address . . . . . : fe80::14aa:991a:5301:eca2%18
   IPv4 Address. . . . . . . . . . . : 10.0.0.1
   Subnet Mask . . . . . . . . . . . : 255.255.0.0
   Default Gateway . . . . . . . . . : ```

**Comment by majestrate** [comment]
we fixed this in #1028
